### PR TITLE
Add invoice lines adjustments derived table

### PR DESCRIPTION
**Pull request by adoerrer**

### CODE REVIEW

File: sql/derived_tables/invoice_lines_adjustments.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
+DROP TABLE IF EXISTS local.rm_invoice_lines_adjustments;
```
Comment (nassibnassar) [inline sql/derived_tables/invoice_lines_adjustments.sql:1]: The `rm_` prefix should not be included in the committed query.

**Comment by nassibnassar** [comment]
What does this comment mean?  "These fields in adjustments can be locally defined"

**Comment by nassibnassar** [comment]
This query does not appear to work in folio_snapshot.  Maybe something has changed in the schema since we loaded most of the data in folio_test_data.

**Comment by adoerrer** [comment]
> 
> 
> This query does not appear to work in folio_snapshot. Maybe something has changed in the schema since we loaded most of the data in folio_test_data.

At this time there are no rows in invoice_lines. I'll ask some aquistions pros to add some invoices to then retest the query.

**Comment by adoerrer** [comment]
> 
> 
> What does this comment mean? "These fields in adjustments can be locally defined"

As I understand institutions can add their own fields in here but I'll try to get more precise information on that and will then change the comment.

**Pull request merged**